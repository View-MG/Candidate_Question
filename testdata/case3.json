[
  {
    "no": 1,
    "platformProductId": "x2-3&FG0A-MATTE-IPHONE16PROMAX*3",
    "qty": 1,
    "unitPrice": 90,
    "totalPrice": 90
  }
]