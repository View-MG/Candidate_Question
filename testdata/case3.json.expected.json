[
  {
    "no": 1,
    "productId": "FG0A-MATTE-IPHONE16PROMAX",
    "materialId": "FG0A-MATTE",
    "modelId": "IPHONE16PROMAX",
    "qty": 3,
    "unitPrice": 30.00,
    "totalPrice": 90.00
  },
  {
    "no": 2,
    "productId": "WIPING-CLOTH",
    "qty": 3,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  },
  {
    "no": 3,
    "productId": "MATTE-CLEANNER",
    "qty": 3,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  }
]